src/main/java/uk/gov/hmcts/reform/managecase/api/errorhandling/RestExceptionHandler.java

```diff
@@ -72,12 +72,6 @@
         return toNoCResponseEntity(HttpStatus.BAD_REQUEST, ex.getErrorMessage(), ex.getErrorCode());
     }
 
-    @ExceptionHandler(CaseIdLuhnException.class)
-    public ResponseEntity<Object> handleCaseIdLuhnException(CaseIdLuhnException ex) {
-        log.error("Invalid caseId : {}", ex.getMessage(), ex);
-        return toResponseEntity(HttpStatus.BAD_REQUEST, ex.getLocalizedMessage());
-    }
-
```

Comment:
Pls restore this one.

Reply:
@smathangi, @mattnayler  - The build didn't fail after the accidental removal of the above method. This tells me that there is no integration or functional tests to challenge any related operation with a non-luhn case id, or CaseIdLuhnException is not actually how we are communicating rejected non-luhn ids from controllers. Am I missing something?

Shall we raise a ticket to add some tests for that, or is there one for it already?

CC: @mario-paniccia .